spec/models/contact_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Contact, type: :model do
+  context 'contact validation test' do
+    it 'contact must have a name' do
+      user = User.create(name: 'Jane', age: 30)
+      contact = Contact.create(name: 'John', user_id: user.id)
```

Comment:
O `contact` não precisa necessáriamente estar salvo no banco nesse teste, e alternativamente ao `user_id` você pode passar a instância de `user` na hora de criar um novo `contact`:

```suggestion
      contact = Contact.new(name: 'John', user: user)
```